styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -1,65 +1,299 @@
+  @include make-xs-column-push(3);
+}
+
+.hostingOptions-hr {
+  border: 0;
+  border-top: 1px solid $gray-light;
+  margin: 0 0 10px;
+}
+
+.hostingOptions-textContainer {
+  @include make-sm-column(12);
+  clear: both;
+}
+
+.hostingOptions-text {
+  margin-bottom: 35px;
+  text-align: center;
+}
+
+.hostingOptions-learnMoreBottom {
+  @extend .bottom;
+}
+
+.hostingOptions-learnMore {
+  color: $brand-primary;
+  font-size: 1.1rem;
+  font-weight: 600;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  &:hover {
+    color: $brand-primary;
   }
 }
 
+
+//solutions
+.solutions {
+  padding: $standard-padding;
+}
+
+.solutions-hoverArea {
+  text-decoration: none;
+
+  &:hover {
+
+    .border-white {
+      border-color: $mustard;
+    }
+
+    h6,
+    .rsweb {
+      color: $mustard;
+    }
+  }
+}
+
+//ctp
+
+.ctp {
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/home/city_imagev2.jpg');
+  background-position: bottom center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.ctp-button {
+  color: $white;
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+
+//quadrant
+
+.quadrant {
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/home/quadrant.png');
+  background-position: 23% bottom;
+  background-repeat: no-repeat;
+  background-size: 35%;
+  padding: 20px 0;
+  padding-bottom: 0;
+}
+
+.quadrant-row {
+  @include make-row;
+}
+
+.quadrant-textContainer {
+  @include make-sm-column(6);
+  @include make-sm-column-push(6);
+}
+
+.quadrant-title,
+.quadrant-text {
+  text-align: right;
+}
+
+.quadrant-buttonContainer {
+  @include make-sm-column(6);
+  @include make-sm-column-push(6);
+  @include make-xs-column(6);
+  @include make-xs-column-push(3);
+  padding: $half-padding-full;
+  text-align: right;
+}
+
+.logoShowcase-image {
+  max-height: 40px;
+  max-width: 75%;
+  text-align: center;
+  vertical-align: middle;
+}
+
+.logoShowcase-row {
+  @include make-row;
+}
+
+.logoShowcase-logoItem {
+  display: inline-block;
+  padding: 1rem .75rem 0;
+  text-align: center;
+}
+
+
+@media only screen and (max-width: $screen-sm-max) {
+
+  .logoShowcase-logoItem {
+    max-width: 100px;
+    padding: 0;
+  }
+
+  .quadrant {
+    background-position: 0% bottom;
+    background-repeat: no-repeat;
+    background-size: 54%;
+  }
+
+}
+
+@media only screen and (min-width: 1850) {
+  .quadrant {
+    background-position: 30% bottom;
+    background-size: 30%;
+  }
+}
+
+
+@media only screen and (max-width: $screen-xs-max) {
+
+  .hostingOptions-product {
+    padding-left: 7px;
+    padding-right: 7px;
+  }
+
+  .quadrant-title,
+  .quadrant-text,
+  .quadrant-buttonContainer {
+    text-align: center;
+  }
+
+  .quadrant {
+    background-image: none;
+  }
+
+  .logoShowcase {
+    display: none;
+  }
+}
+
+@media only screen and (max-width: 600px) {
```

Comment:
I'd say something like $rackspace-break-1 and $rackspace-break-2.  Something that indicates these are rackspace-specific breakpoints.